docs/releasing.md

```diff
@@ -1,16 +1,30 @@
 # Release Process
 
-1. Review and discuss any potential modifications to the [support matrix/policy][support-policy]
-2. Create a draft release in github
-3. Tag the repository and push the tag
+Before beginning the release process, review and discuss any potential modifications to the [support matrix/policy][support-policy].
+
+## Semi-automatic
+
+1. Make sure your repo is clean by git's standards
```

Comment:
Requesting a full stop//period at the end of every sentence here.

Reply:
Periods are only required in bulleted lists when the bullet includes multiple sentences.